src/model.py

```diff
@@ -349,6 +375,45 @@ async def generate(self, request):
             raise e
         finally:
             self.ongoing_request_count -= 1
+            
+    def verify_loras(self, requests):
+        verified_requests = []
+        for request in requests:
+            # We will check if the requested lora exists here, if not we will send a 
+            # response with `LoRA not found` information. In this way we may avoid
+            # further processing.
+            lora_error = None
+            lora_name = None
+            parameters_input_tensor = pb_utils.get_input_tensor_by_name(
+                request, "sampling_parameters"
+            )
+            if parameters_input_tensor:
+                parameters = parameters_input_tensor.as_numpy()[0].decode("utf-8")
+                sampling_params_dict = self.get_sampling_params_dict(parameters)
+                lora_name = sampling_params_dict.pop("lora_name", None)
+            
+            if lora_name is not None:
+                if not self.enable_lora:
+                    lora_error = pb_utils.TritonError("LoRA feature is not enabled.")
+                    self.logger.log_info("[vllm] LoRA is not enabled, please restart the backend with LoRA enabled.")
+                elif lora_name not in self.supported_loras:
+                    lora_error = pb_utils.TritonError(
+                        f"LoRA {lora_name} is not supported, we currently support {self.supported_loras}"
+                    )
+                    self.logger.log_info(f"[vllm] LoRA {lora_name} not found.")
+                
+            if lora_error is not None:
+                output_tensor = pb_utils.Tensor(
+                    "text_output", np.asarray(["N/A"], dtype=self.output_dtype)
```

Comment:
should we send an actual error back, instead on "N/A" ?

Reply:
Apology for my ignorance, but in this case error info has been sent by response_sender:

```python
response_sender.send(
    response, flags=pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL
)
```

and we can receive that:

```
Encountered error while processing: LoRA wizard is not supported, we currently support ['alpaca', 'wizardLM']
```

An error has been sent. I don't know what you mean by modifying `N/A`. I have re-write this code to 

```python
output_tensor = pb_utils.Tensor(
    "text_output",
    np.asarray(["[Error] Unsupported LoRA."], dtype=self.output_dtype),
)
```

If I misunderstood your suggestion, please let me know.

Reply:
Apologies for not being clear. Thanks for changing "N/A", This is nice